bedwars-plugin/src/main/java/com/andrei1058/bedwars/levels/internal/PlayerLevel.java

```diff
@@ -33,21 +34,19 @@
 @SuppressWarnings("WeakerAccess")
 public class PlayerLevel {
 
-    private UUID uuid;
+    private static final ConcurrentHashMap<UUID, PlayerLevel> levelByPlayer = new ConcurrentHashMap<>();
```

Comment:
why is this concurrent?

Reply:
I think that's was touched by "reformat code" tool by Intellij. In the current code it is concurrent. I must have missed it when sending the pr

![image](https://user-images.githubusercontent.com/63428864/152575589-e7a50ae1-27bb-407a-9498-e48323b270b6.png)